content/docs/user-guide/data-management/data-versioning.md

```diff
@@ -0,0 +1,108 @@
+# Data Versioning
+
+DVC enables [version control] for data science. But DVC does not actually
+implement versioning directly! Instead, DVC focuses on [codifying your data]:
+generating small [metafiles] that you can handle with standard [Git workflows]
+(commits, branching, pull requests, etc.).
+
+![Versioned ML project](/img/versioned-project.png) _Navigate versions with Git
+commits_
+
+The resulting projects are neatly organized in the "space dimension", having
+only the files and directories needed at the time and without complicated, ad
+hoc file names like `2022-10-20_linear-model_v2-Carl`. Project versions live in
+the "time dimension" ([Git history]).Here's a more concrete example:
+
+<cards>
+
+<card heading="Traditional">
+
+```cli
+data_v0/      300M
+├── file_1
+├── ...
```

Comment:
Note that the use of `<cards>` here is not final, an image may be better (though I like that there's actual text using cards, for SEO).

> Another option are tabs but then it's harder to see the contrast.